app/templates/components/file-uploader.hbs

```diff
@@ -88,7 +88,7 @@
                         </form>
 
                         <div class="row text-center m-v-md">
-                            <p> {{t "components.preprint-form-project-select.initiate_preprint_process"}} </p>
+                            <p class="finalize-upload-language"> {{t "components.preprint-form-project-select.initiate_preprint_process"}} </p>
```

Comment:
This seems to fix it- but I'm a little surprised that text was ever falling outside the container at all.

Two notes:
1. I think this might be worth a smidge further investigating as to the root cause; otherwise we're likely to keep one-off fixing small instances.

2. Currently, this repo imports osf styles, which define margin and padding helper classes. For example, the div on line 90 uses `m-v-md` (15 px vertical margins). Consider using an applicable pre-defined style helper rather than adding your own, where possible.
https://centerforopenscience.github.io/osf-style/style.html#layout-helper